harvard-lts/drs-import-management-service
# Python/Flask code base

**JIRA Ticket**: ([link](https://app.zenhub.com/workspaces/hdc-62163f60fb08c60011fa4795/issues/harvard-lts/hdc/13))

# What does this Pull Request do?

Adds the Python/Flask application code base.

The application is dockerized and provides a basic /health endpoint.

# How should this be tested?

- Execute the application by creating a container through docker-compose: `docker-compose up` (Docker commands can also be used as an alternative as described in README.md)
- Access http://localhost:13880/health to verify that endpoint returns OK response

# Test coverage
- unit tests? Yes
- integration tests? No
